lib/net.js

```diff
@@ -1485,7 +1497,9 @@ Server.prototype.listen = function() {
     return this;
   }
 
-  throw new Error('Invalid listen argument: ' + util.inspect(options));
+  throw new errors.TypeError('ERR_INVALID_OPT_VALUE',
+                             'listen options',
```

Comment:
IMHO this should be just `options` so it will be rendered as:
```text
The value "{ gaga: 4 }" is invalid for option "options"
```
which seems a tiny bit better then
https://github.com/nodejs/node/blob/77b02d2d8345c407dc0e0121420d823b2ce6fb18/test/parallel/test-net-server-listen-options.js#L63